### PR TITLE
Nav & form

### PR DESCRIPTION
**Pull request by amartya-rs**
feature update: navigation & form components.

### CODE REVIEW

**Review by archit7448 — APPROVED** [review]
looks good to merge

File: all-components/form.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+}
+
+.form-footer {
+    display: flex;
+    align-items: center;
+    gap: 8px;
+}
+
+.form-footer a {
+    font-size: 0.7rem;
```
Comment (bhushannaikaroskar) [inline all-components/form.css:46]: use px here

File: all-components/navigation.css [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+
+/*main navigation styling*/
+.main-nav {
+    padding: 0 1.5rem;
+    background: var(--PRIMARY-COLOR);
+    width: 100%;
+    height: 4.5rem;
+    box-shadow: 0 0 10px 5px var(--GREY);
+
```
Comment (bhushannaikaroskar) [inline all-components/navigation.css:11]: as its 10px I think using 1rem would be better

**Review by bhushannaikaroskar — APPROVED** [review]
Add comments for changes but rest all I think its good to go.

**Review by purvasharmadev — APPROVED** [review]
Everything looks good to merge.

File: all-components/form.css (hunk 1 above)
Comment (amartya-rs) [reply all-components/form.css:46]: I want the font to increase when the user changes the default browser font. That's why used rem.

File: all-components/navigation.css (hunk 2 above)
Comment (amartya-rs) [reply all-components/navigation.css:11]: My root font size if 16px, that's why I used 10px.

**Pull request merged**